INCIDENTS.md

```diff
@@ -0,0 +1,48 @@
+# Incident Response Protocol
+
+
+## What's an incident?
+
+An incident is defined here as a severe immediate or short-term potential degredation in the user-facing performance of our services.
+
+Some examples might be:
+
+- Traffic to `wellcomecollection.org` or some part of the site not being served as expected, e.g. "Whats on?" being unavailable or incorrect
+- Search APIs being unavailable or serving incorrect data
+- A security compromise such as credentials being exposed on the public internet.
+
+An incident must be worked on as soon as possible to mitigate loss of service to users and minimise reputational damage.
+
+## Reporting an incident
+
+An potential incident might be reported in lots of ways (tweet from a user, report from a developer) but should be funneled to a delivery manager to assess the damage and decide whether to call the issue an "incident" or prioritise work via other channels.
+
+There will be situations where it is obvious that a situation needs to be delat with immediately and doesn't need "approval" by a delivery manager to be called an incident, in that situation ensure a delivery manager is informed as part of inititiating an incident.
```

Comment:
I am unclear about this - assuming this is saying that anyone can initiate the protocol? If that's right would something like this work?

```suggestion
There are some situations where a person decides that it is best to initiate the protocol themselves without a delivery manager at first. When this happens, ensure that the delivery manager is informed that this has happened. Some examples of where this might happen

* The incident is small enough to get a rapid fix out
* A delivery manager is unavailable
```